homeassistant/components/lacrosse_view/config_flow.py

```diff
@@ -0,0 +1,128 @@
+"""Config flow for LaCrosse View integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from lacrosse_view import LaCrosse, Location, LoginError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("username"): str,
+        vol.Required("password"): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> list[Location]:
+    """Validate the user input allows us to connect."""
+
+    api = LaCrosse(async_get_clientsession(hass))
+
+    try:
+        await api.login(data["username"], data["password"])
+
+        locations: list[Location] = await api.get_locations()
```

Comment:
This is another example of an unnecessary type hint if the underlying library is fixed

Reply:
I managed to add the `options.package_data` to LaCrosse, but now it says `Returning Any from function declared to return "List[Sensor]"` instead of `List[Any]`.

Reply:
I don't see package data in the `setup.cfg` in the github repo.. I was expecting to see something like this:

```
[options.package_data]
lacrosse_view = py.typed
```

Reply:
Ah sorry, forgot to commit that. I promise it's in the latest release of the PyPI package

Reply:
OK you may want to try to get mypy working on a local change within the library e.g. a test to narrow down the scope of problems.  Just a guess, but is it from the decorator which says that it returns `Any`? I'm not really sure.

When reviewing PRs that update dependencies from pypi, it is required to include a link to the changelog/commits included. The most common and simple approach is to use some of the suggested python github actions that can publish to pypi when you tag a new release in github.

Reply:
Can you please also tag a release on github?

Reply:
Done! https://github.com/IceBotYT/lacrosse_view/commits/v0.0.9